contracts/src/v0.8/dev/OptimismCrossDomainGovernor.sol

```diff
@@ -0,0 +1,69 @@
+// SPDX-License-Identifier: MIT
```

Comment:
Should these contracts be moved outside `/dev` at this point?

Reply:
I'm not actually sure, since other contracts like some `VRF` contracts and the `ArbitrumValidator` are in `dev/`.
More generally, we can reorganize the dev folder in a separate PR.

Reply:
No problem to do it in a separate PR. 
Please ping relevant stakeholders to approve, but I'd assume that's the right move.